src/FixedDataTableStore.js

```diff
@@ -12,9 +12,16 @@
 'use strict';
 
 import reducers from 'reducers';
-import { createStore } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
 
-const store = createStore(reducers);
+const store = configureStore({
+  reducer: reducers,
+  middleware: getDefaultMiddleware => getDefaultMiddleware({
+    serializableCheck: false,
```

Comment:
State contains some non-serializable data, hence have to disable this check

Reply:
It might be worth ticketing out switching to only serializable state.

Reply:
Created a ticket for it
https://jira.schrodinger.com/browse/SS-32511